helpers/validators.js

```diff
@@ -26,7 +26,7 @@ const emailValidator = {
   errorMessage: 'Not a valid email',
   isValid: (email) =>{
     const str = email.toLowerCase();
-    const regexp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    const regexp = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
This wouldn't allow [email subadressing](https://en.wikipedia.org/wiki/Email_address#Subaddressing) (aka "plus sign (+) _trick_"), i.e. `housing-repairs+online@lincoln.gov.uk`

The following suggestion would allow email subaddressing:
```suggestion
    const regexp = /^\w+([\.-]?\w+)*([\+\.-]?\w+)?@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```
...additional tests should need to be added.